IBAnimatableTests/RotationDesignableTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  RotationDesignableTests.swift
+//  IBAnimatable
+//
+//  Created by Steven on 4/30/17.
+//  Copyright © 2017 IBAnimatable. All rights reserved.
+//
+
+import XCTest
+@testable import IBAnimatable
+
+final class RotationDesignableTests: XCTestCase {
+
+  override func setUp() {
+    super.setUp()
+  }
+
+  override func tearDown() {
+    super.tearDown()
+  }
+
+  func testRotate() {
+    helper_testRotate(element: AnimatableView())
+    helper_testRotate(element: AnimatableScrollView())
+    helper_testRotate(element: AnimatableImageView())
+    helper_testRotate(element: AnimatableStackView())
+    helper_testRotate(element: AnimatableLabel())
+    helper_testRotate(element: AnimatableSlider())
+  }
+
+  private func helper_testRotate(element: MockRotationDesignable) {
+    element.rotate = -360
+    XCTAssertEqual(element.transform, .identity)
+    element.rotate = 360
+    XCTAssertEqual(element.transform, .identity)
+    element.rotate = 90
+    let mockTransform = CGAffineTransform(rotationAngle: .pi * 90 / 180)
+    XCTAssertEqual(element.transform, mockTransform)
+  }
+}
+
+// MARK: - MockRotationDesignable
+
+protocol MockRotationDesignable: class, RotationDesignable {
```

Comment:
You got my point. I think the main point to do mixing tests is your mock, it shouldn't be too complicated with your architecture (besides the complexity of the test itself). Saying that I'm wondering if we should put out each mock protocol (in their own files) to make easier to find them when we will start to work on mixing tests.

Since they are internal, it will work as expected, but it may be harder to find them in the project when we will need them in other context.